_posts/2021-02-18-survey/survey.Rmd

```diff
@@ -0,0 +1,314 @@
+
+```{r, eval = TRUE, echo = FALSE, layout="l-body-outset", fig.cap="Number of users reporting to use DL in industry. Smaller groups not displayed."}
+knitr::include_graphics("images/dl_ind.png")
+
+```
+
+In academia, dominant fields (as per survey participants) are bioinformatics, genomics, and IT, followed by biology, medicine, pharmacology, and social sciences:
+
+```{r, eval = TRUE, echo = FALSE, layout="l-body-outset", fig.cap="Number of users reporting to use DL in academia. Smaller groups not displayed."}
+knitr::include_graphics("images/dl_acad.png")
+```
+
+What application areas matter to larger subgroups of "our" users? Nearly a hundred (of 138!) respondents said they used DL for some kind of image processing application (including classification, segmentation, and object detection). Next up was time series forecasting, followed, in third place, by unsupervised learning.
+
+This is the point where I start mixing in questions and comments. The popularity of unsupervised DL comes a bit unexpected; had we anticipated this, we would have asked for more detail here. So if you're one of the people who selected this -- or if you didn't participate, but do use DL for unsupervised learning -- please let us know a bit more in the comments!
+
+Next, NLP is about on par with the former; followed by DL on tabular data and anomaly detection. Bayesian deep learning, reinforcement learning, recommendation systems, and audio processing are still mentioned frequently.
+
+```{r, eval = TRUE, echo = FALSE, layout="l-body-outset", fig.cap="Applications deep learning is used for. Smaller groups not displayed."}
+knitr::include_graphics("images/dl_appl.png")
+```
+
+### Frameworks and skills
+
+We also asked what frameworks and languages participants were using for deep learning, and what they were planning on using in the future. Single-time mentions (e.g., deeplearning4J) are not displayed.
+
+```{r, eval = TRUE, echo = FALSE, layout="l-body-outset", fig.cap="Framework / language used for deep learning. Single mentions not displayed."}
+knitr::include_graphics("images/dl_lang.png")
+
+```
+
+An important thing for any software developer or content creator to investigate concerns proficiency / levels of expertise present in their audiences. It (nearly) goes without saying that expertise is very different from *self-reported* expertise. I'd like to be very cautious, then, to interpret the below results.
+
+While re R skills[^3], the aggregate self-ratings look plausible (to me), I would have guessed a slightly different outcome re DL. Judging from other sources (like, e.g., GitHub issues), I tend to suspect more of a bimodal distribution (a far stronger version of the bimodality we're already seeing, that is). To me, it seems like we have rather many users who know a *lot* about DL. In agreement with my gut feeling, though, is the bimodality itself -- as opposed to, say, a Gaussian shape.
```

Comment:
"with regard", that's good

but no, I didn't mean "regarding" - I meant what I always thought was ablative of Latin _res_ - perhaps it isn't, and I was assuming wrong all the time??? I often saw people write _re:,_ with a colon, but because of my assumption of what it meant, I thought the colon really shouldn't go there ...